CMakeLists.txt

```diff
@@ -73,6 +76,13 @@ option(BUILD_SHARED_LIBS "Build shared (.so, .dylib, .dll) libraries" ON)
 
 set(GINKGO_CIRCULAR_DEPS_FLAGS "-Wl,--no-undefined")
 
+if(GINKGO_BENCHMARK_ENABLE_TUNING)
+    # In this state, the tests and examples cannot be compiled without extra
+    # complexity/intrusiveness, so we simply disable them.
+    set(GINKGO_BUILD_TESTS OFF)
```

Comment:
I think this should be mentioned in `BENCHMARKING.md`.

Reply:
I actually really do not want to document this feature as this benchmark feature is really for experts (i.e. us) if we need to tune some of our own kernels. Benchmark.md is also used for reproducing our paper results and so on, I think it would just confuse people if we talked about tuning or something there.